docs/csharp/fundamentals/program-structure/index.md

```diff
@@ -19,6 +19,29 @@ You can also create a static method named [`Main`](main-command-line.md) as the
 
 In that case the program will start in the first line of `Main` method, which is `Console.WriteLine("Hello world!");`
 
+## Expressions and statements
+
+C# programs are built using **expressions** and **statements**. Expressions produce a value, and statements perform an action:
```

Comment:
Per the style guide, I think expressions and statements should be italicized, not bolded.